BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -566,19 +569,55 @@ - (BOOL)cleanUpBackgroundSessionIfPossibleGivenUserID:(NSString * _Nonnull)userI
     if (backgroundSessionID != nil && [backgroundSessionID isEqualToString:[self backgroundSessionIdentifier]] == NO) {
         //this background session is not self.backgroundSession, most likely we are running inside the main app
         //and this background session is from an extension, clean it up if it has no more pending tasks
+
```

Comment:
This method strikes me as a good candidate for early returns. Several nested if statements with reversed cleanup is harder to read, imo.

Reply:
done